.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## 🙌 Thanks For Making PR
+
+<!--
```

Comment:
People will not remove this heading. I don't want most the PRs we get start with "🙌 Thanks For Making PR", so let's move this into the comment.
```suggestion
<!--
🙌 Thank you for making this PR!

```